public/css/theme.css

```diff
@@ -70,15 +70,15 @@ body {
 p#name a{
   bottom: 8px;
   color: #007ba8;
-  font-family: arial, helvetica;
+  font-family: arial, helvetica, sans-serif;
```

Comment:
Is this part of the stylelint?

Reply:
Yes, having a generic fallback for the font family.